package.json

```diff
@@ -1,46 +1,61 @@
     "lint:js": "jest --config jest.eslint.config.js",
     "test": "cross-env NODE_ENV=development jest --config jest.test.config.js",
-    "test:watch": "yarn test --watch",
+    "test:watch": "pnpm test --watch",
     "test:package": "cross-env NODE_ENV=development ./scripts/test-package",
     "pretest:integration": "./scripts/install-npm-packages-for-integration-tests",
     "test:integration": "cross-env NODE_ENV=development jest --config integration-tests/jest.test.config.js",
-    "format": "yarn format:js && yarn format:md",
+    "format": "pnpm format:js && pnpm format:md",
     "format:js": "prettier --write '{packages,integration-tests,types}/**/*.{js,ts}'",
     "format:md": "prettier --parser markdown --write '**/*.md'",
-    "test:coverage": "yarn build && npm test -- --no-watchman --coverage",
-    "test:ci": "yarn test --ci --no-watchman --maxWorkers=10",
-    "test:coverage:ci": "yarn test:ci -- --coverage && codecov",
+    "test:coverage": "pnpm build && npm test -- --no-watchman --coverage",
+    "test:ci": "pnpm test -- --ci --no-watchman --maxWorkers=10",
+    "test:coverage:ci": "pnpm test:ci -- --coverage && codecov",
     "docs:clean": "rimraf _book",
     "docs:prepare": "gitbook install",
-    "docs:build": "yarn docs:prepare && gitbook build && cp logo/apple-touch-icon.png _book/gitbook/images/apple-touch-icon-precomposed-152.png && cp logo/favicon.ico _book/gitbook/images",
-    "docs:watch": "yarn docs:prepare && gitbook serve",
-    "docs:publish": "yarn docs:clean && yarn docs:build && ./scripts/publish-docs",
-    "typecheck": "yarn typecheck:flow && yarn typecheck:ts",
+    "docs:build": "pnpm docs:prepare && gitbook build && cp logo/apple-touch-icon.png _book/gitbook/images/apple-touch-icon-precomposed-152.png && cp logo/favicon.ico _book/gitbook/images",
+    "docs:watch": "pnpm docs:prepare && gitbook serve",
+    "docs:publish": "pnpm docs:clean && pnpm docs:build && ./scripts/publish-docs",
+    "typecheck": "pnpm typecheck:flow && pnpm typecheck:ts",
     "typecheck:flow": "flow check --include-warnings",
-    "typecheck:ts": "lerna run typecheck:ts",
+    "typecheck:ts": "pnpm --recursive run typecheck:ts",
     "package-readme-entry": "./scripts/package-readme-entry.sh",
     "changeset": "changeset",
     "changeset:version-and-format": "changeset version && prettier --write --parser json '**/package.json'"
   },
-  "workspaces": [
-    "integration-tests",
-    "packages/*"
-  ],
-  "dependencies": {
+  "resolutions": {
+    "graceful-fs": "4.2.8",
+    "y18n": "3.2.2",
+    "lodash": "4.17.21",
+    "handlebars": "4.7.7",
+    "growl": "1.10.5",
+    "dot-prop": "6.0.1",
+    "merge": "2.1.1",
+    "npm-user-validate": "1.0.1",
+    "serialize-javascript": "6.0.0",
+    "diff": "5.0.0",
+    "object-path": "0.11.8",
+    "npm": "6.14.17",
+    "set-value": "4.0.1",
+    "chalk": "3.0.0"
+  },
+  "packageManager": "pnpm@9.15.6",
+  "devDependencies": {
```

Comment:
Semi important: root package.json files should not contain deps but devDeps. We use things in there for development.